boards/nxp/frdm_mcxw71/frdm_mcxw71.dts

```diff
@@ -65,15 +65,20 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		code_partition: partition@0 {
-			reg = <0x0 DT_SIZE_K(896)>;
-			label = "code";
-			read-only;
+		boot_partition: partition@0 {
+			reg = <0x0 DT_SIZE_K(64)>;
+		};
+
+		slot0_partition: partition@10000 {
+			reg = <0x10000 DT_SIZE_K(416)>;
```

Comment:
Partition sizes are not optimal, but we can optimize them later.

Reply:
So optimal sizes would be 416 KiB for the first sector, and 424 KiB for the second sector?

Reply:
I believe the extra sector needs to be in the slot 0 partition. And both need to be large enough for the status area to live in its own sector (which is a bug, hopefully to get fixed some day).